orchestra/project_api/api.py

```diff
@@ -29,6 +30,61 @@ def get_project_information(project_id):
     }
 
 
+def get_projects_information(project_ids):
```

Comment:
Let's have only one function `get_project_information` that accepts a list of ids as an argument.